### DISCUSSION TITLE
Integrate and update Pods core with "WP_Query improvement" in new WordPress 6.1?

### DISCUSSION THREAD

**Discussion by datgausaigon** [pods-framework/pods#6934] [Q&A] [answered]
Hi, 
The next release of WordPress 6.1 have WP_Query improvement. 
Do you have any plan Integrate and update Pods core with this? 

_“WordPress 6.1 includes an improvement to how database queries are performed in the ‘WP_Query’ class so that the result of database queries will be cached in object caching.

This means that if the same database query is run more than once, the result will be loaded from cache.

For those who are using persistent object caching, this will mean that until caches are invalidated, the database query will not be run again, resulting in far few queries to the database.”_

Thanks

**Comment by datgausaigon** [comment]
Some update from make.wordpress.org

[https://make.wordpress.org/core/2022/10/07/improvements-to-wp_query-performance-in-6-1/](https://make.wordpress.org/core/2022/10/07/improvements-to-wp_query-performance-in-6-1/)

**Comment by datgausaigon** [comment]
Hi, new update about this in WordPress 6.1.
https://make.wordpress.org/core/2022/10/11/performance-field-guide-for-wordpress-6-1/

**Comment by sc0ttkclark** [comment, marked as answer]
This performance improvement is automatic and will improve performance across all post-based storage that Pods uses for it's lookups. No changes necessary 🎉

**Reply by datgausaigon** [reply]
Thanks for your reply
